lib/kotkowo_web/live/news_live/found_home.html.heex

```diff
@@ -0,0 +1,162 @@
+<.section class="space-y-10">
+  <.breadcrumb>
+    <:crumb navigate={~p"/aktualnosci"}>
+      Aktualnosci
+    </:crumb>
+    <:crumb navigate={~p"/aktualnosci/znalazly-dom"} active>
+      Znalazły dom
+    </:crumb>
+  </.breadcrumb>
+  <div class="mx-6 lg:mx-0">
+    <h1 class="text-primary mt-6 lg:mt-0 text-justify lg:mt-0 mb-6 font-manrope font-extrabold xl:text-4xl text-3xl">
+      Znalazły dom
+    </h1>
+    <h2 class="font-manrope font-bold text-2xl lg:text-base lg:text-xl">
+      Poznaj kotki, które znalazły już nowych opiekunów.
+    </h2>
+
+    <div class="flex flex-col lg:flex-row gap-x-32 mt-10 lg:mt-20 gap-y-10">
+      <div class="lg:w-80 w-full shrink-0 border border-gray rounded-2xl flex flex-col lg:max-w-none max-w-sm self-center space-y-6 h-min ">
+        <form phx-change="filter_cats" class="p-6 flex flex-col space-y-6 h-min">
+          <div class="flex flex-col space-y-4">
+            <h3 class="font-bold text-primary text-sm">Imię</h3>
+            <input
+              id="catSearch"
+              phx-debounce="100"
+              name="cat_search"
+              type="text"
+              value={@name}
+              placeholder="Wpisz imię kota"
+              class="border border-gray rounded-lg text-sm"
+            />
+          </div>
+
+          <div class="flex flex-col space-y-4">
+            <h3 class="font-bold text-primary text-sm">Tagi</h3>
+            <input
+              type="text"
+              value={@tag}
+              name="tag_search"
+              phx-debounce="100"
+              id="tagSearch"
+              placeholder="Wpisz cechę kota"
+              class="border border-gray rounded-lg text-sm"
+            />
+          </div>
+          <div class="flex flex-col space-y-4">
+            <h3 class="font-bold text-primary text-sm">Płeć</h3>
+            <div class="flex flex-col space-y-6">
+              <div :for={sex <- Sex.all()}>
+                <input
+                  type="checkbox"
+                  name={sex |> Atom.to_string() |> String.downcase()}
```

Comment:
Phoenix will automatically convert atoms to strings for you.
There's also no need to downcase since the atom is already using only lower case characters.

So this can be just:
```suggestion
                  name={sex}
```